src/plugins/validation.ts

```diff
@@ -132,12 +130,59 @@ const isBetween = (
  * @returns {function(*): (boolean|string)}
  */
 const isDateValid = (
-  message?: string
+    message?: string
 ): ((v: string) => string | true | undefined) => {
   message = message || `Invalid Date`;
   // validate date isn't something like 12/DD/YYYY
   return (v: string) => {
-    return (/^[0-9]*$/.test(v.replaceAll(/\//g, ""))) || message 
+    return (/^[0-9]*$/.test(v.replaceAll(/\//g, ""))) || message
+  };
+};
+
+/**
+ * Validator that validates if input is a valid Email
+ * Returns the error message otherwise.
+ *
+ * @param (string) email as "test@mail.mil"
+ * @returns {function(*): (boolean|string)}
+ */
+const isEmail = (): ((v: string) => string | true | undefined) => {
+  return (v: string) => {
+    if (v !== "") {
+      if (/[a-z0-9]+@[a-z]+\.[a-z]{3}/.test(v)) {
+        return "Please use standard domain format, like ‘@mail.mil’"
+      } else if (v.indexOf("@") < 0) {
+        return "Please include an '@’ symbol in the email address."
+      } else if (/[a-z0-9]+@[a-z]+\.[ gov,mil ]/.test(v)) {
+        return "Please use your .mil or .gov email address."
+      }
+    }
+    return true;
+  };
+};
+
+/**
+ * Validator that validates if input is a valid Phone Number
+ * Returns the error message otherwise.
+ *
+ * @param (string) country code
+ * @returns {function(*): (boolean|string)}
+ */
+const isPhoneValid = (country: string, phoneNumber: string
+): ((v: string) => string | true | undefined) => {
+  return (v: string) => {
+    if (v !== "") {
+      if (country == 'us' && v.length < 10) {
+        return "Phone number must be 10 digits including the area code."
+      } else if (country == 'dsn' && v.length == 7 || country == 'dsn' && v.length == 10) {
+        return `DSN number must be 7 digits for CONUS or 10 digits for OCONUS
+         including the geographical code.`
+      } else if (v.length < 7 || v.length > 20) {
+        return `Phone number must be between 7 and 20 digits. It may contain left 
+        and right parentheses, hyphen, period, plus, and spaces.`

```

Comment:
- [ ] user should be allowed to enter `It may contain left and right parentheses, hyphen, period, plus, and spaces`